client/package.json

```diff
@@ -31,7 +31,8 @@
         "prettify-check": "prettier --config ./.prettierrc --list-different \"src/{app,environments}/**/*{.ts,.js,.json,.css,.scss}\"",
         "prettify-write": "prettier --config ./.prettierrc --write \"src/{app,environments}/**/*{.ts,.js,.json,.css,.scss}\"",
         "cleanup": "npm run prettify-write; npm run lint-write",
-        "cleanup-win": "npm run prettify-write & npm run lint-write"
+        "cleanup-win": "npm run prettify-write & npm run lint-write",
+        "generate-defaults": "ts-node -s -r tsconfig-paths/register cli/generate-settings-defaults.ts && npm run prettify-write"
```

Comment:
I was not sure whether to automatically run prettify after generation, I'm open to removing it again if someone does not want to have it here

Reply:
yes, please remove. You could import prettier in your `.ts` files and run it after generating a file directly after file generation only, but right now you would lint and write everything twice if generating both files, which seems unnecessary.